packages/components/src/ProgressBar/_docs/ProgressBar.stickersheet.stories.tsx

```diff
@@ -25,57 +25,94 @@ export default {
   },
 } satisfies Meta
 
+const moods = [
+  {
+    title: "Positive",
+    props: { mood: "positive" },
+  },
+  {
+    title: "Informative",
+    props: { mood: "informative" },
+  },
+  {
+    title: "Negative",
+    props: { mood: "negative" },
+  },
+  {
+    title: "Cautionary",
+    props: { mood: "cautionary" },
+  },
+] satisfies Array<{ title: string; props: Partial<ProgressBarProps> }>
+
+const colors = [
+  {
+    title: "Blue",
+    props: { color: "blue" },
+  },
+  {
+    title: "Green",
+    props: { color: "green" },
+  },
+  {
+    title: "Red",
+    props: { color: "red" },
+  },
+  {
+    title: "Yellow",
+    props: { color: "yellow" },
+  },
+] satisfies Array<{ title: string; props: Partial<ProgressBarProps> }>
+
 const StickerSheetTemplate: StickerSheetStory = {
-  render: ({ isReversed = false }) => (
-    <StickerSheet className="w-full" isReversed={isReversed}>
-      <StickerSheet.Body>
-        <StickerSheet.Row rowTitle="Positive" rowTitleWidth="100px">
-          <ProgressBar
-            value={25}
-            max={100}
-            mood="positive"
-            isAnimating={false}
-            label="25%"
-            subtext="Subtext"
-            isReversed={isReversed}
-          />
-        </StickerSheet.Row>
-        <StickerSheet.Row rowTitle="Informative">
-          <ProgressBar
-            value={25}
-            max={100}
-            mood="informative"
-            isAnimating={false}
-            label="25%"
-            subtext="Subtext"
-            isReversed={isReversed}
-          />
-        </StickerSheet.Row>
-        <StickerSheet.Row rowTitle="Negative">
-          <ProgressBar
-            value={25}
-            max={100}
-            mood="negative"
-            isAnimating={false}
-            label="25%"
-            subtext="Subtext"
-            isReversed={isReversed}
-          />
-        </StickerSheet.Row>
-        <StickerSheet.Row rowTitle="Cautionary">
-          <ProgressBar
-            value={25}
-            max={100}
-            mood="cautionary"
-            isAnimating={false}
-            label="25%"
-            subtext="Subtext"
-            isReversed={isReversed}
-          />
-        </StickerSheet.Row>
-      </StickerSheet.Body>
-    </StickerSheet>
-  ),
+  render: ({ isReversed = false }) => {
+    const defaultProps = {
+      label: "25%",
+      value: 25,
+      max: 100,
+      isAnimating: false,
```

Comment:
Suggestion (although perhaps create a constant instead of calling the function multiple times):
```suggestion
      isAnimating: isChromatic(),
```

Reply:
It's a stickersheet only for chromatic, maybe not worth ever setting it to true?

Reply:
Moreso just for manually checking for yourself :)

But oops, just realised I meant the opposite 😂 It was supposed to be `!isChromatic()`, so that we disable animation for Chromatic, but can see it ourselves.

Reply:
Added an extra example for this instead of making them all animate on local